recipes/psi4/conda_build_config.yaml

```diff
@@ -0,0 +1,23 @@
+channel_sources:
+  - conda-forge/label/libint_dev,conda-forge
+# note for review: I'd engage the following since the src isn't a proper upstream release,
+#   but I don't think it's enough to get this recipe into branch `dev` upon feedstock creation
```

Comment:
It should be enough

Reply:
Thanks! Lines applied, and CI passing again.